veps/sig-network/decouple-nad.md

```diff
@@ -0,0 +1,183 @@
+# VEP 106: Decouple net-attach-def from KubeVirt 
+
+## Release Signoff Checklist
+
+Items marked with (R) are required *prior to targeting to a milestone / release*.
+
+- [x] (R) Enhancement issue created, which links to VEP dir in [kubevirt/enhancements] (not the initial VEP PR)
+- [x] (R) Target version is explicitly mentioned and approved
+- [x] (R) Graduation criteria filled
+
+## Overview
+
+**Target Version**: v1.8
+
+KubeVirt utilizes Network Attachment Definition (NAD) annotations to enforce node-level network resource requirements. When a NAD is annotated with a resource name, KubeVirt automatically adds its value as a resource request to the virtual machine's virt-launcher pod. This mechanism ensures the Kubernetes scheduler only places the VM on nodes that can provide the required network.
+
```

Comment:
Please consider giving SR-IOV, bridge network when bridge-marker [1] is used and macvtap [2] as examples of known use-cases.

[1] https://github.com/kubevirt/bridge-marker
[2] https://github.com/kubevirt/macvtap-cni

Reply:
Please resolve this thread.